### DISCUSSION TITLE
R Studio – Same plot always shows up

### DISCUSSION THREAD

**Discussion by madonnasniffer** [epinowcast/epinowcast#408] [Q&A]
Hi everybody

In my R Studio, I once plotted a certain function. I deleted R and R Studio and reinstalled it. I made a new Working Directory, cleared the session, looked out for unwanted things in the WD, inspected search paths, checked for auto loading scripts and searched for hidden files. If im plotting a function now, still the once plotted functions shows up. I don't know how this is possible and need help.
<img width="496" alt="Bildschirm­foto 2023-11-30 um 21 54 34" src="https://github.com/epinowcast/epinowcast/assets/152555381/29410ea5-d303-4c07-aaa8-17444b88813e">

Mike

**Comment by seabbs** [comment]
Hi @madonnasniffer 

This seems unrelated to using `epinowcast` so I suggest making a post on https://community.rstudio.com/ as they will more likely be able to help. 

Sam
